receiver/sqlqueryreceiver/README.md

```diff
@@ -30,6 +30,7 @@ Each _metric_ in the configuration will produce one OTel metric per row returned
 * `aggregation` (optional): only applicable for `data_type=sum`; can be `cumulative` or `delta`; defaults to `cumulative`.
 * `description` (optional): the description applied to the metric.
 * `unit` (optional): the units applied to the metric.
+* `static_attributes` (optional): static attributes applied to the metrics
```

Comment:
What is the benefit of using this option instead of just using the transform processor?

Reply:
Please review answer above in thread.

Reply:
Yeah, I asked the same Q here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11868#discussion_r912239391

Reply:
Thanks, that answers my question indeed!